docs/API.md

```diff
@@ -73,44 +73,25 @@ IrresponsibleModule:
   enabled: false
 ```
 
+Reek will load this file automatically by default. If you want to load the
+configuration explicitely, you can use one of the methods below.
+
 You can now use either
 
 ```Ruby
 Reek::Configuration::AppConfiguration.from_path Pathname.new('config.reek')
 ```
 
-but you can also pass a hash via `Reek::Configuration::AppConfiguration.from_map`.
-
-This hash can have the following 3 keys:
-
-1.) directory_directives [Hash] for instance:
-
-```Ruby
-  { Pathname("spec/samples/three_clean_files/") =>
-    { Reek::Smells::UtilityFunction => { "enabled" => false } } }
-```
-
-2.) default_directive [Hash] for instance:
-
-```Ruby
-  { Reek::Smells::IrresponsibleModule => { "enabled" => false } }
-```
-
-3.) excluded_paths [Array] for instance:
-
-```Ruby
-  [ Pathname('spec/samples/two_smelly_files') ]
-```
+but you can also pass a hash with the contents of the `config.reek` YAML file
+to `Reek::Configuration::AppConfiguration.from_hash`.
 
-Given the example above you should load that as "default directive" which means that it will
-be the default configuration for smell types for which there is
-no "directory directive" (so a directory-specific configuration):
+Given the example above you should load that as follows:
 
 ```Ruby
 require 'reek'
 
-default_directive = { Reek::Smells::IrresponsibleModule => { 'enabled' => false } }
-configuration = Reek::Configuration::AppConfiguration.from_map default_directive: default_directive
+config_hash = { 'IrresponsibleModule' => { 'enabled' => false } }
```

Comment:
How about adding a "directory directive" and an "exclude path" to the config_hash here so the example is actually comprehensive?